src/app/premier-league/fpl-live/page.tsx

```
@@ -705,31 +942,35 @@ export default function FPLLivePage() {
-                    <MdRefresh className="w-5 h-5" />
-                    <span className="hidden xs:inline sm:inline">
+                    <MdRefresh className="w-4 h-4 sm:w-5 sm:h-5" />
+                    <span className="text-xs sm:text-sm">
                       {t("fplLive.refresh")}
                     </span>
                   </button>
 
                   <button
                     onClick={() => setShowSettings(!showSettings)}
-                    className="flex items-center gap-2 px-4 py-2.5 bg-gray-500 hover:bg-gray-600 text-white rounded-lg text-sm font-medium transition-all min-h-[44px]"
+                    className="flex items-center justify-center gap-2 px-3 py-2.5 bg-black dark:bg-white text-white dark:text-black rounded-lg text-sm font-medium transition-all min-h-[44px] hover:bg-black/90 dark:hover:bg-white/90 theme-transition"
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)